r1cs/gadgets/std/src/fields/mod.rs

```diff
@@ -678,10 +678,39 @@ mod test {
         }
     }
 
+    /*
+    progress test for the inverse gadget, should fail on old, insecure gadget
```

Comment:
Probably here it's better to say "Regression test for inversion gadget in quadratic extension fields..." and then give more details about the purpose of the test ?